vector/src/main/scala/geotrellis/vector/io/package.scala

```diff
@@ -19,9 +19,7 @@ package geotrellis.vector
 import geotrellis.vector.io.wkb.WKB
 import geotrellis.vector.io.wkt.WKT
 
-package object io extends io.json.Implicits
```

Comment:
These extensions have to be removed, because other wise if you do something like this:

```scala
import geotrellis.vector._
import geotrellis.vector.io._
```

The compiler will throw an error about not being able to find an implicit.